File: Data Structures & Algorithms/buy-and-sell-crypto/submission-1.java
class Solution {
    public int maxProfit(int[] prices) {

        int maxProfit = 0;

        for(int i = 0; i < prices.length - 1; i++){
            int cost = prices[i];

            for(int j = i + 1; j < prices.length; j++){

                int sellPrice = prices[j];
                int profit = sellPrice - cost;

                maxProfit = Math.max(maxProfit, profit);
            }
        }

        return maxProfit;
    }
}

File: Data Structures & Algorithms/best-time-to-buy-and-sell-stock-ii/submission-1.js
class Solution {
    /**
     * @param {number[]} prices
     * @return {number}
     */
    maxProfit(prices) {
        let max = 0;

        for(let i = 1; i < prices.length; i++){
            let profit = prices[i] - prices[i - 1];
            if(profit > 0) {
                max += profit;
            }
        }

        return max;
    }
}
